pkg/cvo/cvo_scenarios_test.go

```diff
@@ -117,7 +117,7 @@ func setupCVOTest(payloadDir string) (*Operator, map[string]runtime.Object, *fak
 			Steps: 1,
 		},
 		"exclude-test",
-		false,
+		"Default",
```

Comment:
unclear to me if `Default` or the empty-string is the default value.  Whichever way we go, can you doc it in at least the `Operator` property declaration?